api/insight/socket.io.go

```diff
@@ -56,6 +66,7 @@ type WebSocketTx struct {
 	Hash     string              `json:"txid"`
 	Size     int                 `json:"size"`
 	TotalOut int64               `json:"valueOut"`
+	Vins     []InsightSocketVin  `json:"vins,omitempty"`
```

Comment:
This really is supposed to be plural (`vins`), while the outputs are singular (`vin`).